From 793a8ed8a9f7657049ebbaddf4dc8837899c66de Mon Sep 17 00:00:00 2001
From: kereee30 <nino_kereselidze@epam.com>
Date: Sun, 25 May 2025 03:41:53 +0400
Subject: [PATCH 2/3] Clarify shopping list

---
 demo_repo/shopping.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/demo_repo/shopping.txt b/demo_repo/shopping.txt
index 14b84f4..5131313 100644
--- a/demo_repo/shopping.txt
+++ b/demo_repo/shopping.txt
@@ -1,9 +1,9 @@
 Shopping List
 
 apples
-bananas
+6 bananas
 yogurt
-milk
+2% milk
 wheat bread
 cereal
 eggs
-- 
2.49.0.windows.1

